DataConnectors/CyberArk Data Connector.json

```diff
@@ -0,0 +1,88 @@
+{
+	"id": "CyberArk",

```

Comment:
All the issues reported in the earlier PR #764 for the data connector is not incorporated - please incorporate those.